Sources/Controllers/SpotifyController.swift

```diff
@@ -0,0 +1,119 @@
+//
+//  SpotifyController.swift
+//  ControlKit
+//
+
+import SpotifyiOS
+import SwiftUI
+
+/// Wrapper for ``SPTAppRemote``.
+public final class SpotifyController: NSObject, ObservableObject {
+    
+    lazy private var remote: SPTAppRemote = {
+        let remote = SPTAppRemote(
+            configuration: SPTConfiguration(
+                clientID: SpotifyConfig.clientID,
+                redirectURL: URL(string: SpotifyConfig.redirectURL)!
+            ),
+            logLevel: .debug
+        )
+        remote.connectionParameters.accessToken = accessToken
+        remote.delegate = self
+        return remote
+    }()
+    
+    @AppStorage("SpotifyAccessToken")
+    private var accessToken: String?
```

Comment:
Something telling me that it's better to be Keychain 😄

Reply:
My Security Journey belt should be revoked 🫨

# 🚓👮‍♂️👮‍♀️
![giphy](https://github.com/user-attachments/assets/a9b23feb-1114-441b-bbe0-2e20c0d0f6e8)

Reply:
[Added a property wrapper](https://github.com/superturboryan/ControlKit/pull/1/commits/0260755a81173dfe8e550ceaff1d8f2b699d59d4) for `@Control.Keychain` 🔐  

Let me know what you think @erikbasargin

Reply:
@erikbasargin  as discussed, we'll allow a "data access object" to be injected into `SpotifyController` for the purposes of saving the access token, and provide a default "UserDefaults DAO" in the init. This should provide the best compromise between security and ease-of-use while debugging  🤝

This default parameter should be documented as _unsafe_ in _production_ code, as sensitive data should not be stored in the keychain! 👮